bambi/models.py

```diff
@@ -266,7 +266,7 @@ def fit(
             using the ``fit`` function.
             Finally, ``"laplace"``, in which case a Laplace approximation is used and is not
             recommended other than for pedagogical use.
-            To use the PyMC numpyro and blackjax samplers, use ``nuts_numpyro`` or ``nuts_blackjax``
+            To use the PyMC numpyro and blackjax samplers, use ``numpyro_nuts`` or ``blackjax_nuts``
```

Comment:
Quick question. What is the reason to change these? Is it because the documentation was wrong?

If that is not the case, and we're changing the implementation, we should add some message as it may break users' existing code.

Reply:
The reason for the change is to match the naming convention in bayeux.

Reply:
I think it's the right thing to do, but right now I would like to have something that doesn't break user existing code. Something along the following lines

```python
if inference_method == "nuts_numpyro":
    inference_method = "numpyro_nuts"
    warnings.FutureWarning("'numpyro_nuts has been replaced by 'nuts_numpyro' and will be removed in the future")
elif inference_method == "nuts_blackjax":
    inference_method = "blackjax_nuts"
    warnings.FutureWarning("'blackjax_nuts' has been replaced by 'nuts_blackjax' and will be removed in the future")
```

what do you think about it?

Reply:
I think this is a good idea. It also gives me time to go through the Bambi docs and any `inference_method` passing `numpyro_nuts`.